<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

  <node name="republish" type="republish" pkg="image_transport" args="compressed in:=/robot_camera/image_rect_color raw out:=/robot_camera/image_rect_color" />


  <!-- <param name="/image_sub_topic" value="" type="string" /> -->

  <node name="object_detection_node" pkg="object_detection" type="object_detection_node" output="screen">
    <param name="engine_filename" type="string" value="yolov5s_v1.engine"/>
    <param name="image_size" type="int" value="416"/>
  </node>

  <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find robot_camera_calibration)/rviz/camera_extrinsics_calibration.rviz" /> -->

</launch>
